features/consent/src/test/java/in/projecteka/jataayu/consent/viewmodel/GrantedConsentViewModelTest.kt

```diff
@@ -93,10 +95,19 @@ class GrantedConsentViewModelTest {
         assertEquals(Success(consentsListResponse), consentViewModel.consentListResponse.value)
     }
 
+
+    @Test
+    fun shouldFilterConsentsAndReturnOnlyGrantedRequestedList() {
+        consentViewModel.filterConsents(consentsListResponse.requests)
+        assertFalse(consentViewModel.grantedConsentsList.value!!.filter { it.status == RequestStatus.REQUESTED || it.status == RequestStatus.DENIED }.count() > 0)
+    }
+
     @Test
-    fun shouldFilterConsents() {
```

Comment:
use backticks for unit test naming to make more expressive. Then, it need not have to be named as 
"should <insert what test does>" since the test name is now more descriptive